src/boss/Holmakefile

```diff
@@ -79,6 +79,25 @@ hol4-base.art: hol4-base.thy hol4-base-unint.art ../opentheory/hol4.int \
 install: hol4-base.thy hol4-base.art 
 	opentheory install --reinstall $<
 
+# This installs all HOL-OT theories (after building) in correct order
```

Comment:
This is not good style.  In particular, it assumes that `Holmake` is in the PATH.  Also, we know that recursive make is not a good idea. I'd much rather see explicit targets listed for the sub-directories.  For example, 
```
install-all: $(patsubst %,%/install,$(dirs))
```
If you need the linearisation implied by the `-j1`s or for logical dependency reasons, that dependency information should be in the relevant directories’ `Holmakefile`s.

Reply:
This is also the wrong place for this.  I don't want (implicit) forward pointers from `src/boss` into material (such as `real` and `probability`) that comes later in the build sequence.

Reply:
I'm going to completely revert these changes in `src/boss/Holmakefile`. Actually the best thing I was having in mind, is to have something at toplevel like `bin/build install-ot` (after a full HOL build) but I don't know how to implement it...